src/clj/athens/self_hosted/clients.clj

```diff
@@ -69,7 +69,7 @@
 ;; Client management API
 (defn add-client!
   ([channel]
-   (add-client! channel true))
+   (add-client! channel false))
```

Comment:
What's going on here lol?

Reply:
that's a client that didn't yet introduce itself.